### ISSUE TITLE
Docs: misplaced image in Getting Started

### ISSUE THREAD

**Issue by kingdonb** [weaveworks/weave-gitops#3292]
> Podinfo details showing Podinfo suspended
is the image alt text for the image that appears to be missing, it's under:

https://docs.gitops.weave.works/docs/getting-started/#suspend-updates

I'm not able to get in and fix this right now, but I wanted to make sure it's not lost track of and gets recorded!

**Comment by kingdonb** [comment]
Beg pardon, I don't see the issue anymore. Must have been a network flake out.
